test/nuxt/plugin.test.ts

```diff
@@ -244,4 +244,24 @@ describe('plugin dependsOn', () => {
       'end B'
     ])
   })
+
+  it('expect B to execute after A, C when B depends on A and C', async () => {
+    const nuxtApp = useNuxtApp()
+    const sequence: string[] = []
+    const plugins = [
+      pluginFactory('A', undefined, sequence),
+      pluginFactory('B', ['A', 'C'], sequence),
+      pluginFactory('C', undefined, sequence)
+    ]
+    await applyPlugins(nuxtApp, plugins)
+
+    expect(sequence).toMatchObject([
+      'start A',
+      'start C',
+      'end A',
+      'end C',
+      'start B',
+      'end B'
+    ])
```

Comment:
```suggestion
    const plugins = [
      defineNuxtPlugin({
        name: 'A',
        async setup () {
          sequence.push('start A')
          sequence.push('end A')
        }
      }),
      defineNuxtPlugin({
        name: 'B',
        dependsOn: ['A', 'C'],
        async setup () {
          sequence.push('start B')
          sequence.push('end B')
        }
      }),
      defineNuxtPlugin({
        name: 'C',
        async setup () {
          sequence.push('start C')
          sequence.push('end C')
        }
      })
    ]
    await applyPlugins(nuxtApp, plugins)

    expect(sequence).toMatchObject([
      'start A',
      'end A',
      'start C',
      'end C',
      'start B',
      'end B',
    ])
```

Reply:
`pluginFactory` is returning plugins that are parallels by default and are waiting 10ms before resolving. The reproduction only contains plugins that are not parallel

Reply:
> The reproduction only contains plugins that are not parallel

Yes, you are right!
But isn't it better to test both cases (parallel and non-parallel case), or is it just enough to test the reproduction in the issue?

Reply:
I think the case should be already tested by the test at line 124, i'm ok with keeping both.

Reply:
I see. I started to think keeping both is a bit redundant I will keep only your suggestion. Thanks.